### ISSUE TITLE
compilertest, jitbuildertest and jitbuilderexample are not compiled on supported platforms

### ISSUE THREAD

**Issue by TianyuZuo** [eclipse/omr#1230]
compilertest has testing code on Linux 390 but it is not compiled in job "Compile and archive release" on Linux 390 platforms in the main build. 

If jitbuildertest and jitbuilderexample are not compiled on Linux_x86 and Linux 390 (assuming they are supported on these two platforms).

**Comment by mstoodle** [comment]
Could you maybe rephrase this issue, @TianyuZuo, assuming it belongs here at all?
 Eclipse OMR does not run Linux 390 builds and does not have any "job" like you named...

**Comment by TianyuZuo** [comment]
Just realize that Eclipse OMR does not run Linux 390, I will close this issue.
